packages/components/src/internal/util/utils.ts

```diff
@@ -779,3 +779,15 @@ export function quoteValueWithDelimiters(value: any, delimiter: string) {
     }
     return '"' + value + '"';
 }
+
+export function arrayEquals(a: string[], b: string[], ignoreOrder = true, caseInsensitive?: boolean): boolean {
+    if (a === b) return true;
+    if (a == null && b == null) return true;
```

Comment:
Is this line necessary?

Reply:
This checks when a = null and b = undefined. Not very pretty, I know...